Next.XP 2.0.0 release (#678)
enonic/nextxp-demo: docs/nextjs-setup.adoc

[source,files]
----
# Provide a unique value which will be used to secure the preview mode
ENONIC_API_TOKEN=mySecretKey

# Enonic app name, must match the name of your app
ENONIC_APP_NAME=com.example.myproject

# Enonic XP projects configuration for different locales
ENONIC_PROJECTS=moviedb/hmdb <1>

# Absolute URL to Content API
ENONIC_API=http://127.0.0.1:8080/site <2>
----
+
<1> XP projects configuration.
More about it in <<i18n#, multi-language>> support.

> <2> The base URL to the content APIs created by Guillotine discussed in <<api-primer#, the API>>, but without project and branch parts.

Reviewer: ```suggestion
<2>Base URL of the Guillotine API discussed in <<api-primer#, the API>>, but without project and branch parts.
```
